### ISSUE TITLE
Does not fit: '2 x M3 30mm hex screw (use original screws)'

### ISSUE THREAD

**Issue by atoomnetmarc** [WINTFOX00/TRIDENT-STEPPER-DECOUPLER#3]
The '2 x M3 30mm hex screw (use original screws)' do not fit, they are about 5 mm too short. I used M3x40, they fit.

![afbeelding](https://github.com/WINTFOX00/TRIDENT-STEPPER-DECOUPLER/assets/48124268/99a3274f-a8de-4c15-be10-199eba3165dd)

**Comment by atoomnetmarc** [comment]
> they are about 5 mm too short.

Replying to myself. Actually M3x35 would also not fit. M3x38 would fit perfect. M3x40 is fine:

![afbeelding](https://github.com/WINTFOX00/TRIDENT-STEPPER-DECOUPLER/assets/48124268/b38d2518-48d3-4b31-8800-1bcdb50eb5c9)

**Comment by mstaniszczak** [comment]
Yes, it can't be oryginal screw. It is screwed in slightly higher and have to be screwed into insert, do deeper than oryginal. M3 40mm is fine - even if is too long it's invisible.
